### PR TITLE
Updated the user group section

### PR DESCRIPTION
**Pull request by Aaen**
Added important when you create your own custom, user groups. You will need remember to set a start node for the user group.

### CODE REVIEW

File: Getting-Started/Data/Users/index.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ By default, the User groups available to new users are **Administrators**, **Wri
 
 ## Creating a User Group
 
+:::note
+When creating a custom user group, remember to set a start node for content for that user group. If you don't assign a start node then the users will not be able to login to the backoffice.
```
Comment (jonat123) [inline Getting-Started/Data/Users/index.md:50]:
```suggestion
When creating a custom user group, remember to set a start node for content for the user group. If you don't assign a start node then users assigned to the user group will not be able to login to the backoffice.
```

**Review by jonat123 — COMMENTED** [review]
Good catch @Aaen, I have made a small suggestion, let me know what you think 🙂

**Comment by Aaen** [comment]
@jonat123 I have updated the PR with the changes that you have suggested.

**Comment by jonat123** [comment]
@Aaen great i will merge it in :D

**Pull request merged**